src/components/LandingHero/styles.js

```diff
@@ -28,7 +28,7 @@ export const Title = styled.h1`
   font-size: 40px;
   font-weight: 500;
   color: #40364d;
-  font-family: BrandonGrotesqueMed;
+  font-family: BrandonGrotesque;
```

Comment:
Are you sure about this change? Styles look different now in some dvc.org places

Reply:
Yes, I'm sure. For `font-weight: 500` will be used `Brandon Grotesque Medium`, i.e. the same font as was before.
https://github.com/iterative/dvc.org/blob/blog-merging/src/components/Page/fonts/fonts.css#L87-L89

Reply:
so, fonts look different. We need to fix this.